src/cpp/src/llm/pipeline.cpp

```diff
@@ -9,10 +9,10 @@
 #include "openvino/genai/llm_pipeline.hpp"
 #include "openvino/genai/perf_metrics.hpp"
 
-#include "llm/pipeline_static.hpp"
 #include "llm/pipeline_stateful.hpp"
 #include "llm/pipeline_continuous_batching_adapter.hpp"
 #include "speculative_decoding/speculative_decoding_impl.hpp"
+#include "llm/pipeline_stateful_npu.hpp"
```

Comment:
I believe we avoided `_npu` in the files here. That's the reason why static shape pipeline was called "static shape" but not "npu" one. Also, why do we need one more, again?